### ISSUE TITLE
Multi Screens 

### ISSUE THREAD

**Issue by mrsimonbennett** [gilbitron/Splashify#3]
Would be nice if you could set which screen to apply the wallpaper to :)

**Comment by gilbitron** [comment]
I completely agree. Multi screen support is something I hope to add in the future. At the moment the module we use to set the wallpaper [doesn't support multiscreen](https://github.com/sindresorhus/wallpaper/issues/5).

**Comment by slightlyannoyed1** [comment]
would love multi screen support.  i have 3 monitors with diff backgrounds.  using this is nice, but i need more. thanks

**Comment by sindresorhus** [comment]
Support for multiple screens (macOS only) was added in version 4 of `wallpaper`.
